custom-ops-matrix-multiplication/pyproject.toml

```diff
@@ -0,0 +1,30 @@
+[project]
+authors = [{ name = "Modular Inc", email = "hello@modular.com" }]
+description = "Custom Operations: An Introduction"
+name = "custom-ops-introduction"
+requires-python = ">= 3.9,<3.13"
+version = "0.1.0"
+
+[build-system]
+build-backend = "hatchling.build"
+requires = ["hatchling"]
+
+[tool.hatch.build.targets.wheel]
+packages = ["."]
+
+[tool.pixi.project]
+channels = [
+    "conda-forge",
+    "https://conda.modular.com/max-nightly",
+    "https://conda.modular.com/max",
+    "https://repo.prefix.dev/modular-community",
+]
+platforms = ["linux-64", "osx-arm64", "linux-aarch64"]
```

Comment:
`osx-arm64` needs to be removed.

Reply:
You'd pointed out the same thing on the last one, but why should we remove that platform support? These examples do build and run on macOS, but they fall back on the CPU path for naive implementations. I know they're aimed at Linux where you'll have GPU support, but they can at least be looked at locally on macOS.

Reply:
It goes against the GPU story. Fine if we want to show on CPU.